zero-code-instrumentation/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History: zero-code-instrumentation
+
+### v0.1.0 / 2025-01-21
```

Comment:
Do you already have a release out for this gem?

Reply:
No, I will change to 0.0.0